### PR TITLE
chore: set max SDK version to <3.0.0

### PR DESCRIPTION
**Pull request by chalin**
cc @kwalrath @kevmoo

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,7 +5,9 @@ dart:
   - stable
 
 dart_task:
-  - test: --platform vm,chrome
+  - test: --platform vm
+    xvfb: false
+  - test: --platform chrome
```
[the remark this answers is not in the document]
Comment (chalin) [reply .travis.yml]: Done

File: test/utils.dart [hunk 2]
```diff
@@ -9,6 +9,7 @@ import 'package:test/test.dart';
 /// with the given [message].
 Matcher throwsStringScannerException(String text) {
   return throwsA(predicate((error) {
+    // ignore: deprecated_member_use
```
Comment (kevmoo) [inline test/utils.dart]: Why not fix this? @chalin ?

Comment (chalin) [reply test/utils.dart]: Because this package spans Dart 1 and Dart 2 so we can't use the new API (`TypeMatcher`).

Comment (kevmoo) [reply test/utils.dart]: ack – thanks for clarifying!

**Pull request merged**